common/test/services/prismic/opening-times.test.ts

```diff
@@ -250,112 +281,66 @@ describe('opening-times', () => {
 
   describe('groupExceptionalVenueDays', () => {
     it('groups exceptional days, so that each day within a group fall within 14 days of the first day', () => {
+      const In2021 = {
+        december21st: closedHours({
+          on: new Date('2021-12-21'),
+          type: 'Bank holiday',
+        }),
+
+        december29th: closedHours({
+          on: new Date('2021-12-29'),
+          type: 'Bank holiday',
+        }),
+      };
+
+      const In2022 = {
+        december28th: closedHours({
+          on: new Date('2022-12-28'),
+          type: 'Christmas and New Year',
+        }),
+
+        december29th: closedHours({
+          on: new Date('2022-12-29'),
+          type: 'Christmas and New Year',
+        }),
+
+        december30th: closedHours({
+          on: new Date('2022-12-30'),
+          type: 'Christmas and New Year',
+        }),
+
+        december31st: closedHours({
+          on: new Date('2022-12-31'),
+          type: 'Christmas and New Year',
+        }),
+      };
+
+      const In2023 = {
+        january1st: closedHours({
+          on: new Date('2023-01-01'),
+          type: 'Christmas and New Year',
+        }),
+      };
+
       const exceptionalDays: ExceptionalOpeningHoursDay[] = [
-        {
-          overrideDate: new Date('2021-12-21'),
-          overrideType: 'Bank holiday',
-          opens: '00:00',
-          closes: '00:00',
-          isClosed: true,
-        },
-        {
-          overrideDate: new Date('2022-12-28'),
-          overrideType: 'Christmas and New Year',
-          opens: '00:00',
-          closes: '00:00',
-          isClosed: true,
-        },
-        {
-          overrideDate: new Date('2022-12-29'),
-          overrideType: 'Bank holiday',
-          opens: '00:00',
-          closes: '00:00',
-          isClosed: true,
-        },
-        {
-          overrideDate: new Date('2021-12-29'),
-          overrideType: 'Bank holiday',
-          opens: '00:00',
-          closes: '00:00',
-          isClosed: true,
-        },
```

Comment:
True story: I stared at these lines for five minutes trying to work out why we had December 29th in here twice, and looking through the code under test to work out if it was doing some clever de-duplication we had to test or something.

Before spotting these are in two separate years.

Hence the per-year name spacing in the new code!